src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet
+{ 
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
```

Comment:
The idea here is only mark auto-generated data with aspnet5 keyword, not everything